Makefile

```diff
@@ -63,14 +63,13 @@ test-dependencies:
 	@pip install -q -r test-requirements.txt
 
 test: test-dependencies test-start-minio ## run tests quickly with the default Python
-	coverage run -m pytest -v || $$(make test-stop-minio)
-	@make test-stop-minio
+	coverage run -m pytest -v; r=$$?; $(MAKE) test-stop-minio; exit $$r
```

Comment:
when a test fails, will this still execute the test-stop-minio? that's what the `||` operator was doing before.

Reply:
yes. Kevin and I tested that scenario. make only evaluates the return code of the last executed statement